test/dma_tests.py

```diff
@@ -542,6 +543,121 @@ def gain_check(uri, classname, channel, param_set, dds_scale, min_rssi, max_rssi
     assert rssi >= min_rssi
     assert rssi <= max_rssi
 
+def hardwaregain(uri, classname, channel, dds_scale, frequency, hardwaregain_low, hardwaregain_high):
```

Comment:
I don't really understand what this does. Does this measure changing in the AGC?

Reply:
yes, i would like to use it for a series of tests for fmcomms2

Reply:
Can you update the description a bit to explain its related to AGC? That way others will be able to leverage it more easily later on.

Reply:
Sorry for the delay, the description was updated. Is the current one appropriate?